src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_FlowAnalysis.cs

```diff
@@ -1285,10 +1297,10 @@ static void F(object? x)
 }";
             var comp = CreateStandardCompilation(source, parseOptions: TestOptions.Regular8);
             comp.VerifyDiagnostics(
-                // (21,27): warning CS8600: Cannot convert null to non-nullable reference.
+                // (21,27): warning CS8625: Cannot convert null literal to non-nullable reference.
```

Comment:
>CS8625 [](start = 36, length = 6)

Why isn't this the `W` warning?

Reply:
Because we're assigning to a field. Member signatures should be updated with `?`.

---
In reply to: [175198414](https://github.com/dotnet/roslyn/pull/25516#discussion_r175198414) [](ancestors = 175198414)